st2common/bin/st2-run-pack-tests

```diff
@@ -38,10 +38,11 @@ PACK_TEST_PYTHON_DEPENDENCIES_NAMES=(
     'mock'
     'pytest-cov'
 )
+# While Stackstorm supports Python 3.8, pytest-cov must be run at a lower version
 PACK_TEST_PYTHON_DEPENDENCIES_VERSIONS=(
     '>7'
     '>=4.0.3'
-    '>=6.1.1'
+    '>=5.0.0'
```

Comment:
I wonder if the GHA workflow to regenerate the lockfile should learn to deal with this file somehow.

This file is not a simple "copy the locked version", so we'd need to be careful to not turn that workflow into a maintenance headache.